frontend/src/style/less/components/Footer.less

```diff
@@ -0,0 +1,11 @@
+.footer {
+  display: flex;
+  align-items: center;
+}
+
+#logo {
+  margin: 18px;
+  height: 40px;
+  width: 63.67px;
+  margin-left: 40px;
+}
```

Comment:
Å bruke "px" for størrelse er ikke anbefalt pga skalerbarhet, anbefaler å endre til "rem".

Reply:
Med mindre dette er noe standard NAV logo